AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessor.swift

```diff
@@ -172,7 +172,7 @@ final class ItemListProcessor: StateProcessor<ItemListState, ItemListAction, Ite
     /// - Parameter item: the item to be moved.
     ///
     private func moveItemToBitwarden(item: AuthenticatorItemView) async {
-        guard await services.configService.getFeatureFlag(.enablePasswordManagerSync),
+        guard await services.authenticatorItemRepository.isPasswordManagerSyncActive(),
```

Comment:
🤔 Isn't the feature flag check necessary to check `services.authenticatorItemRepository.isPasswordManagerSyncActive()`?

Reply:
the `isPasswordManagerSyncActive()` check actually checks **both** the feature flag and if the user has turned sync on. So it handles both checks in one call.